lambda/writePass/index.js

```diff
@@ -346,7 +347,7 @@ exports.handler = async (event, context) => {
       parkName: parkName,
       mapLink: parkData[0].mapLink,
       parksLink: parkData[0].bcParksLink,
-      ...(await getPersonalizationAttachment(parkName, facilityName, registrationNumber.toString()))
+      ...QRData
```

Comment:
Reason for this change? Not sure this extra step is necessary.

Reply:
I made the change when debugging something and didn't change back. Going to change it back.